### PR TITLE
Uffizzi Logout

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: lib/uffizzi/cli/login.rb [hunk 1]
```diff
@@ -41,11 +40,10 @@ def handle_failed_response(response)
     end
 
     def handle_succeed_response(response)
-      unless account_valid?(response[:body][:user][:accounts].first)
-        puts 'No account related to this email'
-        return
-      end
-      account_id = response[:body][:user][:accounts].first[:id]
+      account = response[:body][:user][:accounts].first
+      return puts 'No account related to this email' unless account_valid?(account)
```
Comment (7R41N33) [inline lib/uffizzi/cli/login.rb:44]: Let's use Uffizzi Shell instead of this

File: lib/uffizzi/cli/login.rb [hunk 2]
```diff
@@ -41,11 +40,10 @@ def handle_failed_response(response)
...
+
+      account_id = account[:id]
```
Comment (7R41N33) [inline lib/uffizzi/cli/login.rb:46]: This method is very simple. So initializing an additional variable here is redundant

Comment (moklidia) [reply lib/uffizzi/cli/login.rb:46]: Ah, I see!

File: lib/uffizzi/cli.rb [hunk 3]
```diff
@@ -4,21 +4,27 @@
 
 module Uffizzi
   class CLI < Thor
-    desc 'version', 'show version'
+    require_relative 'cli/common'
+
+    class_option :help, type: :boolean, aliases: '-h'
```
Comment (7R41N33) [inline lib/uffizzi/cli.rb:9]: Can it conflict with the `login` command?

```
method_option :hostname, required: true, aliases: '-h'
```

**Pull request merged**